_posts/en/newsletters/2024-10-11-newsletter.md

```diff
@@ -0,0 +1,301 @@
+release candidates.*
+
+- [Bitcoin Inquisition 28.0][] is the latest release of this
+  [signet][topic signet] full node designed for experimenting with
+  proposed soft forks and other major protocol changes.  The updated
+  version is based on the recently released Bitcoin Core 28.0.
+
+- [BDK 1.0.0-beta.5][] is a release candidate (RC) of this library for
+  building wallets and other Bitcoin-enabled applications.  This latest
+  RC "enables RBF by default, updates the bdk_esplora client to retry
+  server requests that fail due to rate limiting. The `bdk_electrum`
+  crate now also offers a use-openssl feature."
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Core Lightning #7494][] introduces a 2-hour lifespan for `channel_hints`,
+  allowing pathfinding information learned from a payment to be reused in future
+  attempts in order to skip unnecessary attempts. Channels that were considered
+  unavailable will be gradually restored and become fully available after 2
+  hours, to ensure that outdated information doesn't cause routes to be skipped
+  that may have since recovered.
+
+- [Core Lightning #7539][] adds a `getemergencyrecoverdata` RPC command to fetch
+  and return data from the `emergency.recover` file. This will allow developers
+  using the API to add wallet backup functionality to their applications.
+
+- [LDK #3179][] introduces new `DNSSECQuery` and `DNSSECProof` [onion
+  messages][topic onion messages], and a `DNSResolverMessageHandler` to handle
+  these messages as the core functionality to implement [BLIP32][]. This PR also
+  adds an `OMNameResolver` that verifies the DNSSEC proofs and turns them into
+  [offers][topic offers]. See Newsletter [#306][news306 blip32].
+
+- [LND #8960][] implements custom channel functionality by adding taproot
+  overlay as a new experimental channel type, which is identical to a [simple
+  taproot channel][topic simple taproot channels] but commits additional metadata
+  in the [tapscript][topic tapscript] leaves for channel scripts. The main
+  channel state machine and database are updated to process and store custom
+  tapscript leaves. A config option `TaprootOverlayChans` must be set to enable
+  support for taproot overlay channels. The custom channels initiative enhances
+  LND’s support for [taproot assets][topic client-side validation]. See
+  Newsletter [#322][news322 customchans].
+
+- [Libsecp256k1 #1479][] adds a [MuSig2][topic musig] module for a
```

Comment:
Ought the Musig optech topic be updated with this entry?

Reply:
I think so. What to take a crack at it?

Reply:
Proposed in https://github.com/bitcoinops/bitcoinops.github.io/pull/1957.